custom/icds_reports/utils/aggregation_helpers/daily_feeding_forms_child_health.py

```diff
@@ -25,21 +25,22 @@ def aggregation_query(self):
 
         return """
         INSERT INTO "{tablename}" (
-          state_id, month, case_id, latest_time_end_processed, sum_attended_child_ids,
-          lunch_count
+          state_id, supervisor_id, month, case_id, latest_time_end_processed,
+          sum_attended_child_ids, lunch_count
         ) (
           SELECT
             %(state_id)s AS state_id,
+            LAST_VALUE(supervisor_id) OVER w AS supervisor_id,
             %(month)s AS month,
-            child_health_case_id AS case_id,
+            DISTINCT child_health_case_id AS case_id,
             MAX(timeend) AS latest_time_end_processed,
```

Comment:
these will need to become `MAX(timeend) OVER w` etc

Reply:
(updated)